plugins/module_utils/core.py

```diff
@@ -188,7 +188,6 @@ def resource(self, service):
         return boto3_conn(self, conn_type='resource', resource=service,
                           region=region, endpoint=ec2_url, **aws_connect_kwargs)
 
-    @property
```

Comment:
won't this break things using `module.region`? Since setting boto3 to False wasn't possible before, maybe the parameter could just be removed?

Reply:
Yeah I think I'll need to do that, I'm poking about now due to a weird failure in ec2_vol_info